api/server/sdk/cloud_backup.go

```diff
@@ -272,6 +282,9 @@ func (s *CloudBackupServer) Delete(
 		CredentialUUID: credId,
 		Force:          req.GetForce(),
 	}); err != nil {
+		if err == api_err.ErrResourceBusy {
+			return nil, status.Errorf(codes.Unavailable, "Resource busy, active Backups in progress")
```

Comment:
should this be `active Deletes in progress` ?

Reply:
yes!. thanks Aditya. Will change that

Reply:
Done.